app/core/check_runs.py

```diff
@@ -0,0 +1,180 @@
+"""
+Set of funtions to create/update/fetch/cancel checkruns
+"""
+
+import logging
+from sqlmodel import select
+
+from app.core.github import PRGithubClient
+from app.helpers.constants import CHECK_RUN_NAME, CheckRunStatus
+from app.helpers.db_utils import DBSession
+from app.helpers.messages import get_cancelled_checkrun_summary
+from app.helpers.utils import get_secret
+from app.models.request_models import PullRequest
+from app.models.sql_models import CheckRun
+
+
+logger = logging.getLogger(__name__)
+
+pr_github_client = PRGithubClient(get_secret("pr-sandbox-pr-installation-id"))
```

Comment:
How about making the naming a bit less redundant?


```suggestion
pr_github_client = PRGithubClient(get_secret("pr-sandbox-installation-id"))
```

Reply:
Yeah, this name turned out to be a bit unfortunate. I don't like it either.

This was actually an attempt at differentiating two different installation ids:
1. `pr-sandbox-pr-installation-id` -  Installation at org where the PRs are located, ex. `openedx`.
2. [pr-sandbox-cluster-installation-id](https://github.com/open-craft/pr-sandbox-automation/blob/c7111d9db45b70f52abc8bbf5a0ffa3c9a8effc9/app/core/sandbox.py#L22) - Installation at org where the cluster is located, ex. `open-craft`.

Perhaps, we name it `pr-sandbox-client-installation-id`. What you think ?